FinSearchUnified/Tests/BusinessLogic/Models/FindologicArticleModelTest.php

```diff
@@ -247,4 +247,63 @@ public function articlesWithAttributeProvider()
             ]
         ];
     }
+
+    public function testArticleKeywords()
+    {
+        $articleConfiguration = [
+            'name' => 'FindologicArticle 1',
+            'active' => true,
+            'tax' => 19,
+            'supplier' => 'Findologic',
+            'categories' => [
+                ['id' => 3],
+                ['id' => 5],
+            ],
+            'images' => [
+                ['link' => 'https://via.placeholder.com/300/F00/fff.png'],
+                ['link' => 'https://via.placeholder.com/300/09f/000.png'],
+            ],
+            'mainDetail' => [
+                'number' => 'FINDOLOGIC1',
+                'active' => true,
+                'inStock' => 16,
+                'prices' => [
+                    [
+                        'customerGroupKey' => 'EK',
+                        'price' => 99.34,
+                    ],
+                ]
+            ],
+            'keywords' => "I'm a simple string,\xC2\xBD,\xC2\x80"
+        ];
+
+        $expectedKeywords = ["I'm a simple string", "\xC2\xBD"];
```

Comment:
Apparently Shopware converts `\xC2\x80` to `\xC3\x82\xC2\x80`. So we end up with `$expectedKeywords` being `["I'm a simple string", "\xC2\xBD", "\xC3\x82"]`. While the last entry makes no sense as a keyword, it is still a valid character.

Reply:
I didn't understand this. How did we know its breaking it into two characters?

Reply:
I saw it during the click test. After Shopware has loaded the keywords it becomes something like `A[]` and after removing the control character all that's left is `A`.

Reply:
@ihanli So should we do the assertion for this extra character or replace it with some other one? because that seems like a garbage character which we have no use of and we are not even setting it technically, so why should we "expect" this character as a keyword?

Reply:
Even if it makes no sense we will end up in the same situation on production, so we are just replicating the scenario.
I would suggest we either add that one character to the expected keywords or use a different one which can be stripped completely. The latter options makes more sense in my opinion but I will leave the decision to you.

Reply:
@ihanli I am not getting this extra character at my setup. 
![image](https://user-images.githubusercontent.com/24492269/63928023-4b491100-ca68-11e9-9ff5-f19be0eaf0f4.png)

Reply:
okay ... :confused: 
Interesting, I can't reproduce it anymore. I will resolve this conversation, so the only thing left is the issue with the assertions.